orchagent/intfsorch.cpp

```diff
@@ -1357,6 +1381,39 @@ void IntfsOrch::doTask(SelectableTimer &timer)
     }
 }
 
+void IntfsOrch::processInbandVlanReady()
+{
+     SWSS_LOG_ENTER();
+
+     Port inbandVlan;
+     if (!gPortsOrch->getInbandPort(inbandVlan))
+     {
+         return;
+     }
+
+     string value;
+     const auto id = sai_serialize_object_id(inbandVlan.m_rif_id);
+     if (m_vidToRidTable->hget("", id, value))
+     {
+         stringstream cmd;
+         cmd << "ip link set " << inbandVlan.m_alias << " up";
+
+         std::string res;
+         int ret = swss::exec(cmd.str(), res);
+         if (ret)
+         {
+             SWSS_LOG_ERROR("Command '%s' failed with rc %d", cmd.str().c_str(), ret);
+         }
+         else
+         {
+             m_stateVlanTable->hset(inbandVlan.m_alias, "state", "ok");
+             SWSS_LOG_INFO("Added inband VLAN %s to STATE_VLAN_TABLE", inbandVlan.m_alias.c_str());
+         }
+
+         m_inbandVlanTimer->stop();
```

Comment:
Do we want to stop the timer when there is failure in bringing up inband vlan in the kernel? Will it be a good idea to leave the timer running for re-tries?

Reply:
Yes, you are right. Fixed.